erde/cli/repl.lua

```diff
@@ -1,6 +1,8 @@
 local C = require('erde.constants')
 local lib = require('erde.lib')
 local utils = require('erde.utils')
+local pack = table.pack or pack
```

Comment:
i think Lua 5.1 has no concept of packing except for table constructors (so this will be `nil`)

Reply:
You're right! I was sure that it has `pack` just like it has `unpack`...